yt_dlp/extractor/mave.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+)
+
+
+class MaveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<channel_id>[a-z]+)\.mave\.digital/ep-(?P<short_id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'https://ochenlichnoe.mave.digital/ep-25',
+            'md5': 'aa3e513ef588b4366df1520657cbc10c',
+            'info_dict': {
+                'id': 'ochenlichnoe-25',
+                'title': 'Между мной и миром: психология самооценки',
+                'description': 'md5:83183d7002dc32fbebc3ccecd4a1ac03',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'ext': 'mp3',
+                'channel': 'Очень личное',
+                'channel_id': 'ochenlichnoe',
+                'channel_url': 'https://ochenlichnoe.mave.digital/',
+            },
+        },
+        {
```

Comment:
to go with the formatting suggestion above:
```suggestion
    }, {
```
and so forth